packages/lib/CalendarService.ts

```diff
@@ -189,21 +189,46 @@ export default abstract class BaseCalendarService implements Calendar {
       }
 
       const eventsToUpdate = events.filter((e) => e.uid === uid);
+      let ev = eventsToUpdate[0];
```

Comment:
```suggestion
      let [event] = eventsToUpdate;
```
NIT: We can try to be more explicit with variable naming.